src/views/StoryViewer.vue

```diff
@@ -19,31 +19,76 @@ export default {
   name: 'StoryViewer',
   data: () => {
     return {
-      frameDuration: 2000,
-      storyEnd: false
+      frameDuration: 6000,
```

Comment:
There starts to be quite a bit of state needed for the progress bar itself. I think it's time to extract it to its own component.